src/shared/ui/form/PasswordRequirements.tsx

```diff
@@ -0,0 +1,103 @@
+import { View } from 'react-native';
+
+import { styled, Text } from '@tamagui/core';
+import { useTranslation } from 'react-i18next';
+
+import { colors } from '@shared/lib/constants/colors';
```

Comment:
Let's put it that way. I promise I'll create a ESLint rule to address the `import` code styles.

```suggestion
import { colors } from '@shared/lib';
```